examples/run_examples_podman/README.md

```diff
@@ -0,0 +1,73 @@
+
+# How to run examples in podman container
+
+This guide walks you through running any of the example in the example directory in podman containers
+
+
+These agents are defined in YAML configuration files located in the `examples/sequential_coordinator` directory.
```

Comment:
s/sequential_coordinator/run_examples_podman

Reply:
that's correct. The run_examples_podman directory have just the container description file, don't have any yaml

Reply:
Ah, got it.